src/test/java/com/jnape/palatable/lambda/adt/choice/Choice3Test.java

```diff
@@ -6,11 +6,7 @@
 import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
-import testsupport.traits.ApplicativeLaws;
-import testsupport.traits.BifunctorLaws;
-import testsupport.traits.FunctorLaws;
-import testsupport.traits.MonadLaws;
-import testsupport.traits.TraversableLaws;
+import testsupport.traits.*;
```

Comment:
Not sure what your intellij config is for lambda. I haven't changed the import settings